docs/docs/topics/schema.mdx

```diff
@@ -14,7 +15,61 @@ Out of the box, Infrahub doesn't have a schema for most things and it's up to us
 
 Unlike traditional databases that can only have one schema at a time, in Infrahub it is possible to have a different schema per branch. This is possible because the schema itself is stored in the database like any other object.
 
-There are several ways to [load a new schema](/guides/import-schema).
+## Schema file
+
+The recommended way to manage and load a schema is to create a schema file in YAML format. With a schema file it's possible to:
+
+- Define new nodes
+- Extend nodes, by adding attributes or relationships to the existing nodes
+
+At a high level, the format of the schema file looks like the following:
+
+```yaml
+---
+version: '1.0'
+nodes:
+    - <new nodes are defined here>
+extensions:
+  nodes:
+    - <node extensions are defined here>
+```
+
+<details>
+  <summary>Example of schema file that is defining new nodes and adding a relationship to an existing one</summary>
+  <CodeBlock language="yaml">{import_infrastructureExtensionRackYaml}</CodeBlock>
+</details>
+
+:::note
+
+To help with the development process of a schema definition file, you can leverage [schema validation](/reference/schema-validation) within your editor.
+
+:::
+
+## Load a schema file
+
+Schema files can be loaded into Infrahub with the `infrahubctl` command or directly via the Git integration. For a detailed explanation and instructions on how to install `infrahubctl`, you can check out our [infrahubctl topic](/infrahubctl).
+
+<!-- vale off -->
+### infrahubctl command
+<!-- vale on -->
+The `infrahubctl` command can be used to load individual schema files or multiple files as part of a directory.
+
+```shell
+infrahubctl schema load <path to schema file or a directory> <path to schema file or a directory>
+```
+
+### Git integration
+
+You can defined a schema in an [external repository](/topics/repository).
+The schemas that should be loaded must be declared in the ``.infrahub.yml`` file, under schemas.
+
+Individual files and directory are both supported.
+
+```yaml
+---
+schemas:
+  - schemas/demo_edge_fabric.yml
+```
```

Comment:
Maybe we can remove the following note below, it's redondant and not really related to the section `load schema`